src/99-appendices/06-meg-file-formats.md

```diff
@@ -180,10 +180,18 @@ Where:
 
 ```Text
 sub-control01_ses-001_task-rest_run-01_meg/
-    sub-control01_ses-001_task-rest_run-01_markers.<mrk,sqd>
+    sub-control01_ses-001_task-rest_run-01[_acq-<label>]_markers.<mrk,sqd>
     sub-control01_ses-001_task-rest_run-01_meg.<con,sqd>
 ```
 
+Marker coil files MUST have an acquisition parameter if there is more than
```

Comment:
I propose to write it as 

If there are multiple files with marker coils, they must have the `acq-<label>` parameter and no more that two files may be associated with one raw data file. While the acquisition parameter can take any value, it is RECOMMENDED that if the two marker measurements occur before and after the raw data acquisition,
 `pre` and `post` are used to differentiate the two situations.